wagtailio/static/sass/components/_hero.scss

```diff
@@ -0,0 +1,61 @@
+.hero {
+    position: relative;
+    grid-column: 2 / span 2;
+    margin-bottom: 50px;
+
+    @include media-query(medium) {
+        grid-column: 2 / span 3;
+    }
+
+    @include media-query(large) {
+        margin-bottom: 100px;
+        grid-column: 2 / span 5;
+    }
+
+    &__inner {
+        padding-top: 20px;
+
+        @include media-query(medium) {
+            padding-top: 40px;
+            max-width: 80%;
+        }
+
+        @include media-query(large) {
+            padding-top: 80px;
+            max-width: 70%;
+        }
+    }
+
+    &__intro {
+        margin-bottom: 20px;
+
+        @include media-query(medium) {
+            margin-bottom: 40px;
+        }
+    }
+
+    &__button {
+        display: inline-flex;
+    }
+
+    &__icon {
```

Comment:
Suggestion: It might be worth extracting some of this into the `icons.scss` file, obviosuly some of this is specific to this element.

Reply:
Feels like the styles used are quite specific to the hero area. We can always pull them out if it becomes apparent they'll be re-used a lot